src/GovUk.Education.ExploreEducationStatistics.Common/Utils/DbContextUtils.cs

```diff
@@ -0,0 +1,54 @@
+#nullable enable
+using System;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Utils
+{
+    public static class DbContextUtils
+    {
+        public static void UpdateTimestamps(object sender, EntityEntryEventArgs e)
+        {
+            switch (e.Entry.State)
+            {
+                case EntityState.Added:
+                {
+                    if (e.Entry.Entity is ICreatedTimestamp entityWithCreated)
+                    {
+                        entityWithCreated.Created = DateTime.UtcNow;
+                    }
+
+                    if (e.Entry.Entity is IUpdatedTimestamp entityWithUpdated)
```

Comment:
Just wondering if there's a particular reason for setting the updated date here on entities that are only just being created?

Looking at the way the date is assigned to both I suppose these two dates could differ by a tiny fraction making it more difficult to report on entities that have been created but not updated yet.